src/extras/BitmapText.js

```diff
@@ -518,29 +518,44 @@ export default class BitmapText extends core.Container
         data.size = parseInt(info.getAttribute('size'), 10);
         data.lineHeight = parseInt(common.getAttribute('lineHeight'), 10) / res;
         data.chars = {};
+        if (textures.constructor !== Array)
```

Comment:
Need to look at how do we check Array instances in other pixi source filtes. I think its `instanceof Array`